content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -25,6 +25,7 @@ The developer preview version provides a preview of the upcoming changes to the
   - [**threads_deleted**](/messaging/agent-chat-api/v3.4/rtm-reference/#threads_deleted)
 - The **List Archives** method has a new filter, `event_types`. It replaces the `events` filter.
 - Multiple methods no longer require the `chats--my:ro`, `chats--my:rw`, `chats.conversation--my:rw`, and `chats.conversation--my:rw` scopes.
+- The **Start Chat**, **Resume Chat**, **Send Event**, **Send Typing Indicator** and **Send Rich Message Postback** methods require `chats--all:rw` or `chats--access:rw` scope instead of `chats.conversation--all:rw` or `chats.conversation--access:rw`.
```

Comment:
```suggestion
- The **Start Chat**, **Resume Chat**, **Send Event**, **Send Typing Indicator**, and **Send Rich Message Postback** methods now require either the `chats--all:rw` or `chats--access:rw` scope instead of the `chats.conversation--all:rw` or `chats.conversation--access:rw`.
```